e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -95,18 +96,34 @@ internal static class TestHelper {
                         var json = await response.Content.ReadAsStringAsync();
                         Assert.NotEmpty(json);
                         var entryModels = JsonConvert.DeserializeObject<PublishedNodesEntryModel[]>(json);
-                        
+
                         Assert.NotNull(entryModels);
                         Assert.NotEmpty(entryModels);
                         Assert.NotNull(entryModels[0].OpcNodes);
                         Assert.NotEmpty(entryModels[0].OpcNodes);
-                        
+
                         result.Add(url, entryModels[0]);
                     }
                 }
             }
-            
+
             return result;
         }
+
+        /// <summary>
+        /// Get configuration that reads from:
+        ///     - .env file
+        ///     - environment variables
+        /// </summary>
+        /// <returns></returns>
+        public static IConfigurationRoot GetConfiguration() {
+            var configuration = new ConfigurationBuilder()
+                .AddFromDotEnvFile()
+                .AddEnvironmentVariables()
+                .AddEnvironmentVariables(EnvironmentVariableTarget.User)
+                .Build();
+
+            return configuration;
+        }
```

Comment:
Please remove we only support Environment variables within e2e test, and each should be loaded manually

Reply:
As discussed, I have done the following:
- Removed configuration provider that loads data from `.env` file.
- Introduced configuration provider that loads environment variables from `launchSettings.json` to enable local debugging.